README.md

```diff
@@ -104,7 +104,7 @@ certain versions of Kotlin.
 
 | Kotlin         | Cite          |
 |----------------|---------------|
-| 2.0.21         | 0.4.0 - 0.6.0 |
+| 2.0.21 - 2.1.0 | 0.4.0 - 0.6.0 |
```

Comment:
I think this is wrong. After #110 I tested the 0.6.1 release and it requires kotlin 2.1.0 or greater:

```
cite-api-jvm-0.6.1.jar!/META-INF/cite-api.kotlin_moduleModule was compiled with an incompatible version of Kotlin. The binary version of its metadata is 2.1.0, expected version is 1.9.0.
```

My project was previously using Cite 0.6.0 with kotlin 1.9.24.

Reply:
Ah, I just forgot to set the languageVersion to remain compatible. The compiler plugin API did not change between these two Kotlin versions, but the metadata version bumped (for Kotlin/Native reasons, so kinda annoying it breaks JVM artifact compatibility, too).

Tomorrow problem.

Reply:
I tried to add a test with our minimum-supported version to actually test the fix, and failed spectacularly. Will try again tomorrow.